### PR TITLE
Added tests for "fmn.rule.update"

### PR DESCRIPTION
**Pull request by Ghost-script**
Fixed Bug: https://github.com/fedora-infra/fedmsg_meta_fedora_infrastructure/issues/247

### CODE REVIEW

**Comment by Ghost-script** [comment]
@ralphbean , Sorry couldn't respond to https://github.com/fedora-infra/fedmsg_meta_fedora_infrastructure/pull/251 on time, due to issues with my internet conection. 

Please review this patch for #247 .

File: fedmsg_meta_fedora_infrastructure/tests/fmn.py [hunk 1]
```diff
@@ -243,6 +243,35 @@ class TestConfirmationUpdateStatus(Base):
         }
     }
 
+class TestRuleUpdate(Base):
+    """ Messages of this type are published whenever **the status of a
+    confirmation changes** in the `Fedora Notifications
```
Comment (pypingou) [inline fedmsg_meta_fedora_infrastructure/tests/fmn.py]: I'm not sure what you mean by ``the status of a confirmation changes`` here, maybe we could rephrase this a bit

Comment (ralphbean) [inline fedmsg_meta_fedora_infrastructure/tests/fmn.py]: Yeah, the text here should be revised to something that reflects the actual message being tested.  Something like ``a rule is updated`` or something similarly vague would be fine.  :)

Comment (Ghost-script) [inline fedmsg_meta_fedora_infrastructure/tests/fmn.py]: Oops, copy paste error, :D , Fixing it.

**Comment by ralphbean** [comment]
:1: to merge, @Ghost-script.  Thanks!

**Pull request merged**